diff -urN serpentine-0.6.3.orig/data/serpentine.desktop.in serpentine-0.6.3/data/serpentine.desktop.in
--- serpentine-0.6.3.orig/data/serpentine.desktop.in	2005-09-18 11:48:38.724657352 +0200
+++ serpentine-0.6.3/data/serpentine.desktop.in	2005-09-18 11:52:05.482225424 +0200
@@ -6,6 +6,6 @@
 Type=Application
 Exec=serpentine
 Icon=gnome-dev-cdrom-audio.png
-Terminal=False
-Categories=GNOME;Application;AudioVideo;X-Ximian-Main;X-Red-Hat-Base;
+Terminal=false
+Categories=GTK;GNOME;AudioVideo;Audio;
 StartupNotify=true
