diff -urN serpentine-0.6.91.orig/data/serpentine.desktop.in serpentine-0.6.91/data/serpentine.desktop.in
--- serpentine-0.6.91.orig/data/serpentine.desktop.in	2006-04-09 11:08:43.000000000 +0200
+++ serpentine-0.6.91/data/serpentine.desktop.in	2006-04-09 11:11:24.000000000 +0200
@@ -6,7 +6,7 @@
 Type=Application
 Exec=serpentine -o %U
 Icon=gnome-dev-cdrom-audio.png
-Terminal=False
+Terminal=false
 Categories=GNOME;GTK;AudioVideo;DiscBurning;
 MimeType=audio/x-scpls;audio/x-mpegurl;
 StartupNotify=true
